eng/Version.Details.xml

```diff
@@ -19,38 +19,38 @@
     </Dependency>
     <Dependency Name="System.Diagnostics.DiagnosticSource" Version="4.6.0-rc1.19453.4" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>5f2c2771e077f98c097ecc1b9202bee22ef50aed</Sha>
```

Comment:
Strange. Suspect this branch needs to be manually fixed because there should be no changes here.